src/classes/Ingredient.js

```diff
@@ -0,0 +1,9 @@
+class Ingredient {
+	constructor(ingredient) {
+		this.id = ingredient.id;
+		this.name = ingredient.name;
+		this.costInCents = ingredient.estimatedCostInCents
+	}
+};
+
+export default Ingredient
```

Comment:
I wonder if we will need to be consistent with export styles. Otherwise looks good for this iteration!